Tests/PointFreeTests/StripeHookTests.swift

```diff
@@ -1,83 +0,0 @@
-import Either
```

Comment:
This file moved to `StripeWebhooksTest.swift`. I always try to quick open typing "web" enough to rename.